velox/exec/tests/HashJoinTest.cpp

```diff
@@ -1202,6 +1202,92 @@ TEST_F(HashJoinTest, antiJoinWithFilter) {
       "SELECT t.* FROM t WHERE NOT EXISTS (SELECT * FROM u WHERE t0 = u0 AND t1 <> u1)");
 }
 
+TEST_F(HashJoinTest, nullAwareAntiJoinWithFilter) {
```

Comment:
There is an existing test named antiJoinWithFilter. One may think that antiJoinWithFilter is testing regular anti join while nullAwareAntiJoinWithFilter is testing null-aware anti join, but both tests use null-aware anti join. Consider combining these tests to avoid confusion.